roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
so this is like this for almost every service, which is why i thought we'd need it. when i deployed on my local environment, it didn't seem to affect the apache2 logrotate file because i think this one was getting saved as /etc/logrotate.d/horizon. either way, now i know apache2 handles logrotate and i can remove this change.